web/src/modules/layout/components/Layout.js

```diff
@@ -9,7 +9,8 @@ import withPlatform from '../../platform/hocs/withPlatform'
 const RichLayout = styled.div`
   position: relative;
   display: flex;
-  min-height: 100vh;
+  height: 100vh;
+  max-height: 100vh;
```

Comment:
You readded `max-height` instead of `min-height` :D

Reply:
🤦🏼